### PR TITLE
Turn on GLM analysis and option to pick observation source

### PR DESCRIPTION
**Pull request by hu5970**
1) Turn on GLM analysis in EnKF
2) Add option (OBSTYPE_SOURCE) to decide which observation resource (rap or rrfs) should be used.
3) update namelist options for smoke and dust.

### CODE REVIEW

File: parm/FV3.input.yml
Comment (MatthewPyle-NOAA) [inline parm/FV3.input.yml:1]: Are these dust/smoke setting changes intentional?

**Review by MatthewPyle-NOAA — COMMENTED** [review]
@hu5970 One small item I wasn't sure about, but otherwise looks good to me.  Thanks for opening this PR.

**Review by MatthewPyle-NOAA — APPROVED** [review]
Giving conditional approval...but was curious what the smoke/dust changes are about.

**Review by BenjaminBlake-NOAA — COMMENTED** [review]
The changes look good to me.  Have they been tested on WCOSS2?  If so I can approve.

Similar to Matt I am also curious about the smoke/dust namelist changes to remove enh_mix, dust_opt, drydep_opt, and dust_alpha.

**Comment by hu5970** [comment]
@MatthewPyle-NOAA @BenjaminBlake-NOAA I should mention that all those changes are tested with the v0.8.3. The changes for smoke/dust matches the current model code. Those changes should be added by Haiqin in his coming PR. But I have to add them to avoid model crash at hour 60.

**Review by BenjaminBlake-NOAA — APPROVED** [review]
@hu5970 Thanks for confirming

**Pull request merged**